src/llmq/quorums_utils.cpp

```diff
@@ -254,5 +257,47 @@ bool CLLMQUtils::IsQuorumActive(Consensus::LLMQType llmqType, const uint256& quo
     return false;
 }
 
+bool CLLMQUtils::IsQuorumTypeEnabled(Consensus::LLMQType llmqType, const CBlockIndex* pindex)
+{
+    const Consensus::Params& consensusParams = Params().GetConsensus();
+    bool f_v17_Active =  VersionBitsState(pindex, consensusParams, Consensus::DEPLOYMENT_V17, versionbitscache) == ThresholdState::ACTIVE;
```

Comment:
does it make sense to combine these lines?

```suggestion
    bool f_v17_Active =  VersionBitsState(pindex, Params().GetConsensus(), Consensus::DEPLOYMENT_V17, versionbitscache) == ThresholdState::ACTIVE;
```

Reply:
Imo current version is slightly more readble